ENGAGEHF/Home.swift

```diff
@@ -66,18 +51,14 @@ struct HomeView: View {
     
     return HomeView()
         .previewWith(standard: ENGAGEHFStandard()) {
-            ENGAGEHFScheduler()
-            MockWebService()
             AccountConfiguration(building: details, active: MockUserIdPasswordAccountService())
         }
 }
 
 #Preview {
```

Comment:
This complete preview could be be deleted as we don't have to differentiate between showing the mock upload or not showing the mock upload.